Linear_cell_complex/include/CGAL/draw_linear_cell_complex.h

```diff
@@ -406,7 +406,7 @@ void draw(const CGAL_LCC_TYPE& alcc,
   {
     int argc=1;
     const char* argv[2]={"lccviewer","\0"};
-    QApplication app(argc,const_cast<char**>(argv));
+    QApplication app(CGAL::code_to_call_before_creation_of_QCoreApplication(argc),const_cast<char**>(argv));
```

Comment:
Do you really need to call the function in the constructor ? 
In other demo it is called before.
(same question for all the draw_XXX functions)

Reply:
I thought so, maybe it has been the case in some older Qt version, but now it is not useful anymore. I started by copying what was done in the polyhedorn demo, and as I had to repete the same operations many times I started to think about it, and changed things  a little.